### PR TITLE
[BAEL-2322] Test Code

### PR DESCRIPTION
**Pull request by psevestre**
Article: http://inprogress.baeldung.com/wp-admin/post.php?post=200696&action=edit&classic-editor
Jira: http://jira.baeldung.com/browse/BAEL-2322

### CODE REVIEW

File: libraries-data-2/pom.xml [hunk 1]
```diff
@@ -216,5 +216,5 @@
             </plugin>
         </plugins>
     </build>
-
+
```
Comment (KevinGilmore) [inline libraries-data-2/pom.xml:219]: Don't need the whitespace change. And this is from a completely different module.

File: aws-reactive/.sts4-cache/classpath-data.json [hunk 2]
```diff
@@ -0,0 +1 @@
...ng\\repo\\tutorials\\aws-reactive\\target\\test-classes","javadocContainerUrl":"file:/C:/Users/Philippe/work/baeldung/repo/tutorials/aws-reactive/target/site/apidocs","isSystem":false,"isOwn":true,"isTest":true,"isJavaContent":true},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\main\\resources","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\classes","isSystem":false,"isOwn":true,"isTest":false,"isJavaContent":false},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\test\\resources","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\test-classes","isSystem":false,"isOwn":true,"isTest":true,"isJavaContent":false},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\test\\java","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\test-classes","javadocContainerUrl":"file:/C:/Users/Philippe/work/baeldung/repo/tutorials/aws-reactive/target/site/apidocs","isSystem":false,"isOwn":true,"isTest":false,"isJavaContent":false}]}
```
Comment (KevinGilmore) [inline aws-reactive/.sts4-cache/classpath-data.json:1]: Don't think we need this file in the PR.

File: persistence-modules/pom.xml [hunk 3]
```diff
@@ -75,6 +75,7 @@
         <!-- <module>spring-mybatis</module> --> <!-- needs fixing in BAEL-9021 -->
         <module>spring-persistence-simple</module>
         <module>spring-persistence-simple-2</module>
+        <module>apache-bookkeeper</module><!-- BAEL-2322 -->
```
Comment (KevinGilmore) [inline persistence-modules/pom.xml:78]: Let's keep the modules in alphabetical order.

Comment (psevestre) [reply persistence-modules/pom.xml]: Done

File: aws-reactive/.sts4-cache/classpath-data.json (hunk 2 above)
Comment (psevestre) [reply aws-reactive/.sts4-cache/classpath-data.json]: Removed

File: libraries-data-2/pom.xml (hunk 1 above)
Comment (psevestre) [reply libraries-data-2/pom.xml]: Removed extra line. For a brief moment, bookkeper was there..

**Pull request merged**